src/Serokell/Util/Group.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE ExplicitForAll      #-}
+{-# LANGUAGE NoImplicitPrelude   #-}
```

Comment:
Could you put `NoImplicitPrelude` in default extensions and clean up this extensions everywhere? Don't you mind to perform extra more refactoring?